### ISSUE TITLE
[bug] Minor issue - "Replace failed node" doesnt ask if the now node should be added securely

### ISSUE THREAD

**Issue by Dinth** [OpenZWave/Zwave2Mqtt#216]
**Version**
2.1.1

Build/Run method
- [x] Docker
- [ ] PKG
- [ ] Manually built (git clone - npm install - npm run build ) 

Zwave2Mqtt version: 2.1.1
Openzwave Version: 1.6

**Describe the bug**
Replace failed node function doesnt open a dialog to add a new node securely, as "Node inclusion" function does.

**Comment by robertsLando** [comment]
@Dinth thanks!

**Comment by robertsLando** [comment]
@Fishwaldo How can I call replace failed node with secure flag enabled??  http://www.openzwave.com/dev/classOpenZWave_1_1Manager.html#a6dc832fe823228a821b81ac900b01c5c

**Comment by Fishwaldo** [comment]
Right now you can’t. Please open a issue on the OpenZWave tracker. 

In those cases, only option now is remove failed node and then add node with secure true.

**Comment by robertsLando** [comment]
@Fishwaldo Yes but this means the new inserted node will take an incremental id instead of taking the id of the replaced one?

**Comment by Dinth** [comment]
> @Fishwaldo Yes but this means the new inserted node will take an incremental id instead of taking the id of the replaced one?

Yes, the newly node will have incrementally increased ID. Which wouldnt be a huge problem (a minor issue, mostly in terms of usability - a new user may expect this option to be there), but i believe at the moment the only way of getting rid of dead nodes is actually replacing them with something else.
